crates/qasm3/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "qiskit-qasm3"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+
+[lib]
+name = "qiskit_qasm3"
+crate-type = ["cdylib"]
+
+[features]
+# This is a test-only shim removable feature.  See the root `Cargo.toml`.
+default = ["extension-module"]
+extension-module = ["pyo3/extension-module"]
+
+[dependencies]
+pyo3.workspace = true
+indexmap.workspace = true
+hashbrown.workspace = true
+oq3_semantics = "0.0.7"
```

Comment:
Does this pin the version at `0.0.7`?  In principle we could make backward compatible bug fixes in 0.0.8. There are certainly some bugs that would affect this importer.

Reply:
It looks like we might want to release `0.1.0` of the parser in order to allow fixes:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

Reply:
Then `oq3_semantics = "0.1.0"` would be compatible with `0.1.1`, etc.

Reply:
Yes, this dependency is equivalent to `>=0.0.7,<0.0.8`. Cargo treats leading zeros specially - it basically treats the leftmost non-zero as the semantic-version "major".  For how we're updating the base library at the moment, I think that's actually fairly accurate (and not a problem on either side).

That said, this doesn't really affect anything, because we build our Rust crates into cdylib Python extensions that we treat give-or-take like fixed binary objects - one of those ways is that we pin our dependencies with the lockfile, so only precise lockfile updates affect the build environment, and (for the most part) are made separately to Rust code changes.  In this case, large tracts of the lockfile are updated because there's so many new dependencies getting pulled in.

Reply:
In any case, I just released openqasm3_parser 0.1.0 ., This is the same as 0.0.7. Just changed the version (and reduced the sleep between publishing crates to 60 s) : https://crates.io/crates/oq3_semantics